src/server/pages/user-page.jsx

```diff
@@ -50,6 +50,8 @@ export const isLoggedIn = userInfo => "name" in userInfo;
 export default class UserPage extends React.Component {
   render() {
     const { thisUser, userInfo, notebookList } = this.props;
+    const isUserAccount =
+      isLoggedIn(userInfo) && thisUser.name === userInfo.name;
```

Comment:
Just out of curiosity, the `name` must be unique but what is the underlying data? Where does it come from?

Reply:
So we use a kind of hacky strategy on the server of embedding this kind of metadata inside the html as a script element. Here's where it comes from in this case:

https://github.com/iodide-project/iodide/blob/0e06560240c21a6665edccee496265408f33e47c/server/views.py#L71

Reply:
side note: i think we should change that hacky implementation detail at some point -- https://github.com/iodide-project/iodide/issues/1416